### PR TITLE
[no ticket][risk=no] Running puppeteer tests in circleci

### PR DESCRIPTION
**Pull request by aweng98**
Running all tests locally have passed okay.
Screen Shot 2020-05-21 at 2 00 35 PM

### CODE REVIEW

File: e2e/puppeteer-custom-environment.ts [hunk 1]
```diff
@@ -21,24 +21,46 @@ class PuppeteerCustomEnvironment extends PuppeteerEnvironment {
...
-        const fileName = `${testName}_${timestamp}.png`
+        const fileName = `${testName}_${timestamp}.png`;
+
         const screenshotFile = `${screenshotDir}/${fileName}`;
         await this.global.page.screenshot({path: screenshotFile, fullPage: true});
-        console.error(`Test "${event.test.name}" failed. Saved screenshot ${screenshotFile}.`);
+        console.error(`Saved screenshot ${screenshotFile}`);
+
+        const htmlFileName = `${testName}_${timestamp}.html`;
+        await this.savePageToFile(htmlFileName);
```
Comment (aweng98) [inline e2e/puppeteer-custom-environment.ts:38]: save both page html and take a screenshot when test fails.

**Comment by als364** [comment]
!!!!

File: e2e/puppeteer-custom-environment.ts (hunk 1 above)
Comment (aweng98) [reply e2e/puppeteer-custom-environment.ts:38]: @als364 This change is it. Others are cleanup and improvement. 🙂Thanks for the quick turnaround.

Comment (als364) [reply e2e/puppeteer-custom-environment.ts:38]: thank you!

**Pull request merged**